test-cases/longevity/longevity-large-partition-4days.yaml

```diff
@@ -2,8 +2,14 @@ test_duration: 6480
 
 bench_run: true
 
-prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -max-rate=300 -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -concurrency=7 -rows-per-request=10"]
-stress_cmd: ["scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -rows-per-request=10 -concurrency=7 -max-rate=32000 -duration=5760m"]
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=20 -clustering-row-count=10000000 -clustering-row-size=2048 -concurrency=50000 -rows-per-request=2000 -timeout=180s -connection-count 10000",
```

Comment:
Reduce the "-connection-count to 1000", I noticed in later tests that there is a problem with too high number like 10K.

Reply:
fixed